File: app/src/main/java/com/nilhcem/blenamebadge/ui/custom/knob/OnCrollerChangeListener.java
package com.nilhcem.blenamebadge.ui.custom.knob;

public interface OnCrollerChangeListener {
    void onProgressChanged(Croller croller, int progress);

    void onStartTrackingTouch(Croller croller);

    void onStopTrackingTouch(Croller croller);
}

File: app/src/main/java/org/fossasia/badgemagic/ui/custom/knob/OnCrollerChangeListener.kt
package org.fossasia.badgemagic.ui.custom.knob

interface OnCrollerChangeListener {
    fun onProgressChanged(croller: Croller, progress: Int)

    fun onStartTrackingTouch(croller: Croller)

    fun onStopTrackingTouch(croller: Croller)
}
